packages/suite/src/views/onboarding/steps/BasicSettings/index.tsx

```diff
@@ -16,9 +16,7 @@ const BasicSettings = () => {
                 <AdvancedSetup>
                     <OnboardingButtonCta
                         data-test="@onboarding/coins/continue-button"
-                        onClick={() => {
-                            goToNextStep();
-                        }}
+                        onClick={goToNextStep}
```

Comment:
Be careful with that, it's not the same. See `goToNextStep` implementation: https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/actions/onboarding/onboardingActions.ts/#L60

It receives one param called `stepId` and checking `if (stepId)`. And by this refactor it will receives event as the param.